tools.yml

```
@@ -2293,6 +2307,13 @@ tools:
     mem: 8
   Extract genomic DNA 1:
     cores: 3
+  __docker_tool:
```

I think it may make sense to rename this to `__docker_only_tool`, as most tools can run with docker. Also, would this run with singularity also. If so, perhaps it should be renamed to container_only_tool?
```suggestion
  __docker_only_tool:
```

In general, I've been wondering about the best way to support docker, especially for scenarios like these where the container hasn't worked and we've had to override it manually: https://github.com/galaxyproject/galaxy-helm/blob/697f744f1258f5efccaf52cc3be79e1177ad79fc/galaxy/values.yaml#L524

---

That's a very good point. These tools all have only a docker type requirement in the <requirements/> tag but I forgot that these tools can also run with singularity, so setting `require: [docker]` is too restrictive, as is setting the docker_enabled param. I don't know if this PR makes sense - perhaps this logic belongs in local configuration.

---

What if we just rename `require: [docker]` to `require: [container]`?
I think this information being in the shared database is useful, because you can't realistically expect to run these tools otherwise?